hack/e2e-dynamicfile.sh

```diff
@@ -54,6 +55,79 @@ function e2e_mountfile() {
 
 }
 
+function e2e_dynamicfile_username_prefix_enforce(){
```

Comment:
No change to this PR, but can you create an issue/AI to follow up and separate the role creation/configuration logic to a separate function? Right now the test is mixed with environment configuration

Reply:
yep sure! will create a separate PR for it